ERP/resources/views/product-manager.blade.php

```diff
@@ -0,0 +1,361 @@
+@extends('layouts.master')
+@section('inside-body-tag')
+
+<div class="container-fluid">
+    <div class="panel panel-primary">
+        <div class="panel-heading">
+            <h3 class="panel-title">Panel title</h3>
+        </div>
+        <div class="panel-body">
+
+            <div class="row">
+                <div class="col-4">
+                    <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#bicycle_modal">
+                        Add a new Bicyle
+                    </button>
+                </div>
+
+                <div class="col-4">
+                    <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#materials_modal">
+                        Materials
+                    </button>
+                </div>
+
```

Comment:
Try to remove unnecessary white spaces just for readability purposes

Reply:
I will do that. Thanks